app/scripts/components/index.jsx

```diff
@@ -47,6 +50,13 @@ const Index = ({ create, sessions, adminRights }) => (
   </div>
 );
 
+Index.formattedDate = date => {
+  const unformattedDate = new Date(date);
+  const locale = "en-gb";
+  const formattedDate = unformattedDate.toLocaleString(locale, { month: "short", year: "numeric", day: "2-digit" });
+  return formattedDate;
+}
```

Comment:
I wonder if this would be better as a helper, 

```js

const locale = 'en-gb';
const defaults = {
  day: '2-digit',
  month: 'short',
  year: 'numeric',
};

export const formattedDate = (date, options) => (
  new Date(date).toLocaleString(locale, { ...defaults, ...options })
)

...

import { formattedDate } from '../helpers/date';

...

formattedDate(created)
```